elasticai/creator/nn/quantized_grads/base_modules/conv2d.py

```diff
@@ -1,65 +1,21 @@
-from typing import Any, Callable, Protocol
+from typing import Any
 
 from torch import Tensor
-from torch.nn import Conv2d as __Conv2d
-from torch.nn.functional import conv2d
+from torch.nn import Conv2d as TorchConv2d
+from torch.nn import Module
+from torch.nn.utils import parametrize as P
 
-from elasticai.creator.base_modules.math_operations import Quantize
-from elasticai.creator.nn.quantized_grads.quantized_parameters import (
-    QuantizationSchemeByName,
-    QuantizedParameters,
-)
 
-
-class _Conv2d(__Conv2d):
-    """
-    This class is a facade for Conv2d to allow clean multiple inheritance for Conv2d.
-    """
-
-    def __init__(
-        self,
-        in_channels: int,
-        out_channels: int,
-        kernel_size: int | tuple[int],
-        stride: int | tuple[int] = 1,
-        padding: int | tuple[int] | str = 0,
-        dilation: int | tuple[int] = 1,
-        groups: int = 1,
-        bias: bool = True,
-        device: Any = None,
-        dtype: Any = None,
-        *args,
-        **kwargs,
-    ):
-        super().__init__(
-            in_channels=in_channels,
-            out_channels=out_channels,
-            kernel_size=kernel_size,
-            stride=stride,
-            padding=padding,
-            dilation=dilation,
-            groups=groups,
-            bias=bias,
-            padding_mode="zeros",
-            device=device,
-            dtype=dtype,
-        )
-
-
-class MathOperations(Quantize, Protocol): ...
-
-
-class Conv2d(_Conv2d, QuantizedParameters):
+class Conv2d(TorchConv2d):
     """This module implements a 2d convolution.
```

Comment:
just a comment: you can skip `"This module"` and just state `"A 2d Convolution."` which tbh, we can tell by the name already. Can you provide a more concrete description here? What's different from torch.nn.Conv2d?